packages/nextjs/components/scaffold-eth/Contract/InheritanceTip.tsx

```diff
@@ -0,0 +1,14 @@
+export const InheritanceTip = ({ inheritedFrom }: { inheritedFrom?: string }) => {
+  return (
+    <>
+      {inheritedFrom && (
+        <span
+          className="tooltip tooltip-top tooltip-secondary px-2 before:content-[attr(data-tip)]"
+          data-tip={`Inherited from: ${inheritedFrom}`}
+        >
+          *
```

Comment:
How about we use `InformationCircleIcon` from hero icons?

And maybe `px-1` would suffice on the span.

Reply:
Wanted to ask the same :)

Reply:
OK, yes, I agree. I've been thinking the same thing, maybe we don't need any padding since the icon is going to be a bit larger. I'll try it out.

Reply:
Fixed here: https://github.com/FilipHarald/scaffold-eth-2-sandbox/commit/365e56d31cfb4f33ae7cf8dfc58a1a9976f3d9b7